### PR TITLE
Add initial uniswap price feed library

### PR DESCRIPTION
**Pull request by mrice32**
This is the initial stub for the Uniswap price feed class. Currently, it only computes the current Uniswap price.

Issue #1138.

### CODE REVIEW

**Comment by mrice32** [comment]
@ptare could you take a first pass?

File: financial-templates-lib/UniswapPriceFeed.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    this.uniswap = new web3.eth.Contract(abi, uniswapAddress);
+  }
+
+  getCurrentPrice = () => this.currentPrice;
+
+  _update = async () => {
+    // TODO: optimize this call. This may be very slow or break if there are many transactions.
+    const events = await this.uniswap.getPastEvents("Sync", { fromBlock: 0 });
+
+    // If there are no prices, return null and allow
```
Comment (ptare) [inline financial-templates-lib/UniswapPriceFeed.js]: Did you mean abort instead of allow?

Comment (mrice32) [reply financial-templates-lib/UniswapPriceFeed.js]: Oh no, I just didn't finish writing the comment haha.

Comment (mrice32) [reply financial-templates-lib/UniswapPriceFeed.js]: Done.

**Pull request merged**